src/jinja2/lexer.py

```diff
@@ -607,22 +612,37 @@ def tokenize(
         name: t.Optional[str] = None,
         filename: t.Optional[str] = None,
         state: t.Optional[str] = None,
+        preserve_comments: bool = False,
```

Comment:
suggestion: We could make it a keyword-only parameter for easier backward-compatibility maintenance in the future.